CHANGELOG.md

```diff
@@ -5,6 +5,32 @@
 
 # Changelog
 
+## [v8.20.0](https://github.com/nextcloud-libraries/nextcloud-vue/tree/v8.20.0) (2024-10-29)
+[Full Changelog](https://github.com/nextcloud-libraries/nextcloud-vue/compare/v8.19.0...v8.20.0)
+
+### 🚀 Enhancements
```

Comment:
I'd add a note to the changelog to make it more noticeable.

````suggestion
## [v8.20.0](https://github.com/nextcloud-libraries/nextcloud-vue/tree/v8.20.0) (2024-10-29)
[Full Changelog](https://github.com/nextcloud-libraries/nextcloud-vue/compare/v8.19.0...v8.20.0)

### 📌 Note

Now you can use `v-model` directive, `:model-value` prop, and `@update:model-value` (`@update:modelValue`) event in all data input components to have **v9 compatibility**.

```html
<!-- Only v8 (Vue 2) -->
<NcTextField :value.sync="username" />

<!-- Compatible with both v8 (Vue 2) and v9 (Vue 3) -->
<NcTextField v-model="username" />
<!-- same as -->
<NcTextField :model-value="username" @update:model-value="username = $event" />
```

If you already use `v-model` with a custom writable computed on `:value.sync` components before, it will be broken. Make sure to remove or adjust it before update.

See the full list in [v9 breaking changes](https://github.com/nextcloud-libraries/nextcloud-vue/blob/next/CHANGELOG.md#-breaking-changes).

### 🚀 Enhancements
````